src/picterra/client.py

```diff
@@ -39,7 +54,10 @@ def validate_detector_args(detection_type: str, output_type: str, training_steps
 
 class APIClient():
     """Main client class for the Picterra API"""
-    def __init__(self, api_key=None, base_url=None):
+    def __init__(
+        self, api_key: str = '', base_url: str = '',
+        timeout: int = 30, max_retries: int = 3, backoff_factor: int = 10
+    ):
```

Comment:
We're just missing doc for the new arguments here. LGTM otherwise

Reply:
Checked